src/Kconfig.coap

```diff
@@ -40,22 +40,54 @@ config GOLIOTH_MAX_NUM_OBSERVATIONS
     help
         The maximum number of CoAP paths which can be simultaneously observed.
 
+choice
+    prompt "Golioth blockwise download: Max block size"
```

Comment:
Let's add a name like `choice GOLIOTH_BLOCKSIZE_DN` to be able to override default value on application-level.